### PR TITLE
Je/print statements

### PR DESCRIPTION
**Pull request by emord**
@nickpell @millerdev 

ran modernize to fix print statements and exception catching.

### CODE REVIEW

**Review by millerdev — APPROVED** [review]
Guessing you want to get this merged ASAP to avoid conflicts. I did not review every line, but it's pretty automatic and I think not much can go wrong with these particular conversions. Assuming you looked at every line that was changed to verify that it made sense.

**Comment by emord** [comment]
@millerdev yeah I did a quick check on each line by doing `git add -p .` I did catch one error, but everything else looked ok

**Comment by nickpell** [comment]
@emord curious, what was the error you found?

File: custom/ilsgateway/views.py [hunk 1]
```diff
@@ -252,7 +252,7 @@ def end_report_run(request, domain):
         rr.complete = True
         rr.has_error = True
         rr.save()
-    except ReportRun.DoesNotExist, ReportRun.MultipleObjectsReturned:
```
Comment (emord) [inline custom/ilsgateway/views.py]: @nickpell this line had the error. Whenever an exception tries to catch two things like this, but doesn't have a variable assocaited with it, it is changed to:

```
except ReportRun.DoesNotExist as xxx_todo_changeme:
    ReportRun.MultipleObjectsReturned = xxx_todo_changeme
```

Comment (nickpell) [inline custom/ilsgateway/views.py]: ah ok.  I've since this one before.

**Pull request merged**